docs/developer/extending/webhooks/synchronous-events/shipping.mdx

```diff
@@ -152,16 +167,24 @@ This is an example response from the application:
 ]
 ```
 
+Where:
+
+- `id` **(required)** is id of external shipping method e.g `dhl-exp`. It will be converted by Saleor to `app:<app-database-id>:<id-returned-by-webhook>` and decoded in Base64 format.
```

Comment:
suggestion: I think there was a mistake here, it will be **encoded** as base64

```suggestion
- `id` **(required)** is id of external shipping method e.g `dhl-exp`. It will be converted by Saleor to `app:<app-database-id>:<id-returned-by-webhook>` and encoded in Base64 format.
```